docs/openapi/components/schemas/common/ExternalIdentifier.yml

```diff
@@ -0,0 +1,49 @@
+$linkedData:
+  term: ExternalIdentifier
+  '@id': https://w3id.org/traceability#ExternalIdentifier
+title: External Identifier
+description: Information to recognize & verify an external identifier.
+type: object
+properties:
+  type:
+    type: array
+    readOnly: true
+    const:
+      - ExternalIdentifier
+    default:
+      - ExternalIdentifier
+    items:
+      type: string
+      enum:
+        - ExternalIdentifier
+  name:
+    title: URI
+    description: Name of identifier.
+    type: string
+    $linkedData:
+      term: uri
+      '@id': https://schema.org/name
+  value:
+    title: URI
+    description: Value of identifier.
+    type: string
+    $linkedData:
+      term: uri
+      '@id': https://schema.org/value
+  uri:
+    title: URI
+    description: URI (Universal Resource Identifier) for identifier.
+    type: string
+    $linkedData:
+      term: uri
+      '@id': https://schema.org/url
+additionalProperties: false  
+required:
+  - type
+example: |-
+  {
```

Comment:
Nit: Can you update the example to use leiCode with a correct URI? 
It'll make it easier for future users to adopt

Reply:
@mkhraisha I updated the example to use a correctly formatted leiCode, but I'm not sure what the standard is for the URI at which the lei code can be looked up. I put down `http://example.gov/lei/`, will that work?

Reply:
yes